pandas/core/generic.py

```diff
@@ -3931,22 +3935,24 @@ def _drop_axis(self, labels, axis, level=None, errors: str = "raise"):
 
         return result
 
-    def _update_inplace(self, result, verify_is_copy: bool_t = True) -> None:
+    def _update_inplace(self, result, verify_is_copy: bool_t = True, **kwargs) -> None:
```

Comment:
**kwargs is not very specific as to what arguments you intend to pass.  It would probably be better to use a better variable name.  You should also clarify what you intend to store in **kwargs so that way others know in the future how to use that method.